### ISSUE TITLE
Garage Door and Floodlight Support

### ISSUE THREAD

**Issue by mschindler504** [shauntarves/wyze-sdk#112]
First off, great work!

Hopefully I didn't overlook this, but I was wondering if there was support for accessing/controlling the accessories that connect to the Cam v3's, specifically garage door, and the floodlight?

**Comment by shauntarves** [comment]
Hi there...thanks for your interest in the library!

I don't currently have any support for these types of devices. I am absolutely willing to add support, but that will generally require [at least temporary] access to the devices so I can debug them and understand the API contract.

If you're willing to share the devices with me for a bit of time via the Wyze app (my account email is shaun <at> tarves <dot> net), I can look into them.

It's also really helpful to know what _kind_ of support you're looking for - that is, what would you like to be able to do with the devices via this library?

Cheers!

**Comment by mschindler504** [comment]
Good deal! 

As for support, for the floodlights, the ability to see current light status (on/off), would be great to toggle the device on/off, but I guess there's no big need to change any settings.

Same concept with the garage door... See the status, open/close the garage (as you can do in the app), but really don't see any need or extra efforts in changing settings/properties.

Will share a floodlight for right now.

Thank you sir!

**Comment by shauntarves** [comment]
Okay, please let me know when you share it. Nothing has come through yet

**Comment by mschindler504** [comment]
Floodlight was shared. Will share the garage in a few hours.

Let me know if you need anything else!

**Comment by shauntarves** [comment]
Okay, I've got the floodlight now. Give me a little time with this. Don't be surprised if I kick it on and off a few times 🤣 

Let me know when you share the garage door.

**Comment by mschindler504** [comment]
Garage Door is shared!

**Comment by wesbos** [comment]
Happy to test this with my garage door as well. Really want to control it with home assistant

**Comment by shauntarves** [comment]
Yeah, it's not looking good for the garage door and the floodlight controls. They seem to be controlled by the same kind of non-standard, TCP socket-based communications as the actual cameras themselves. I'm going to keep poking around, but I'm not confident I can get control of these.

**Comment by shauntarves** [comment]
Can either of you view these cams on the Wyze Web View (https://view.wyze.com)? If so...I'd be reallllly curious if you can control the light/door from there. Please let me know.

**Comment by mschindler504** [comment]
I'm able to view the Live stream, but there's no other functionality with the camera other than the audio input/output. 

@shauntarves, I mentioned that the garage door can be controlled with Alexa and Google Assistant, but I did notice that this is only possible via verbal commands to the assistants. There isn't an actual button/toggle within either assistant apps to control the door.

**Comment by wesbos** [comment]
I poked around at all the requests on the wyze live view website and there isn’t anything related to controlling the garage door.

**Comment by shauntarves** [comment]
I played around with Alexa and Google Home a bit yesterday, but the way that both of these services work is that the calls from the client merely go to amazon's or google's services, which then handle making the appropriate request to the wyze service. There's no way to see where those calls are going.

**Comment by mschindler504** [comment]
I had a feeling that would be the case. 

You ever done anything with any of the Google APIs?
